content/docs/setup/kubernetes/additional-setup/config-profile/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Installation Configuration Profiles
+description: Describes the built-in Istio installation configuration profiles.
+weight: 35
+keywords: [profiles]
```

Comment:
Maybe a few more keywords to improve "see also". Look at other files for any keywords that . would make sense here.

Reply:
install,helm

Reply:
Done